front/src/hooks/useAuthorization.ts

```diff
@@ -1,10 +1,14 @@
 import { useAuth0 } from "@auth0/auth0-react";
 import { JWT_ROLE, JWT_ABP, JWT_BETA } from "utils/constants";
 
+/**
+ * requiredAbps: [abp1, abp2] => user must have abp1 AND abp2
+ * 
+ * @todo Do users have such ABP structure? 
+ * requiredAbps: [[abp1, apb2], abp3] => user must have (abp1 OR abp2) AND abp3
```

Comment:
@fhenrich33 it's not about the users having such an ABP structure but about the menues sometimes requiring _at least_ one of several ABPs instead of _all_ of the listed ABPs.
So yes, for flexibility we need the implementation here the way it was before.